src/styles/base.css

```diff
@@ -1,43 +1,43 @@
+:root {
+  --main-purple: #8424e2;
+  --light-purple: #a29ab6;
+  --hot-blue: #1a4bff;
+  --blue: #361ee0;
+
+  --background-purple: #160f29;
+  --secondary-background-purple: #292044;
+  --lightest-background-purple: #50426c; /* use white text */
+
+  --border-lightest-background: #71678a; /* border color. use with lightest background purple */
+
+  --text-headers: #fff;
+  --text-white: #fff;
+  --text-normal: #ffffffcc; /* cc = 80% */
+
+  /* these rgb color values should be identical to their pairs above */
+  /* use these to change opacity like this: `rgba(var(--main-purple-rgb), 0.8)` */
+  --main-purple-rgb: 132, 36, 226;
```

Comment:
you're great @DianaPerkinsDesign. thanks for the useful comments and the nice consolidation cleanup here.